html/js/utils.js

```diff
@@ -67,35 +67,44 @@ ut.getName = function(path, withoutExtension) {
 };
 
 /*
- * Replace the same unit URL as my unit URL with personium-localunit
+ * Convert to "personium-localunit" only when the Cell (cellUrl) is in the same Personium Unit (unitUrl).
+ * Core only recognizes the new format (single slash): personium-localunit:{cellName}:/{path}
+ * Example (Cell URL): "personium-localunit:dixonsiu:/"
+ * Example (Box URL): "personium-localunit:dixonsiu:/app-calendar-box/"
  */
 ut.changeUnitUrlToLocalUnit = function (cellUrl, cellName, unitUrl) {
     var result = cellUrl;
     if (unitUrl.startsWith(cm.user.baseUrl)) {
-        result = ut.PERSONIUM_LOCALUNIT + "/" + cellName + "/";
+        result = ut.PERSONIUM_LOCALUNIT + cellName + ":/";
     }
 
     return result;
 };
 
 /*
- * Replace personium-localunit with your unit URL
+ * Convert "personium-localunit:{cellName}:/" to the normal URL format when the Cell (cellUrl) is in the same Personium Unit.
+ * Cell URL: personium-localunit:/dixonsiu  (supporting old format)
+ *           personium-localunit:/dixonsiu/ (supporting old format)
+ *           personium-localunit:dixonsiu:/ new
```

Comment:
We should probably support the following also
personium-localunit:dixonsiu:

Reply:
It would not be critically necessary.  Just to  make it more friendly.

Reply:
I add unofficial support for missing ending slash. Please check.